DOCSP-44237 flags Option deprecation on v2.21 (#201)
mongodb/docs-php-library: source/reference/method/MongoDBDatabase-createCollection.txt

         :manual:`db.createCollection()
         </reference/method/db.createCollection>` for more information.

>          *Deprecated*: This option is deprecated and will be removed in 
>          {+php-library+} v2.0. For more information on upgrading from the MMAPv1 storage 

Reviewer: ```suggestion
         *Deprecated*: This option is deprecated and will be removed in 
         the v2.0 {+library-short+} release. For more information on upgrading from the MMAPv1 storage 
```

>          {+php-library+} v2.0. For more information on upgrading from the MMAPv1 storage 
>          engine to Wired Tiger, see the 
>          :manual:`Change a Self-Managed Standalone to WiredTiger </tutorial/change-standalone-wiredtiger/>` 
>          guide.

Reviewer: ```suggestion
         {+php-library+} v2.0. To learn more about upgrading from the MMAPv1 storage 
         engine to Wired Tiger, see the 
         :manual:`Change a Self-Managed Standalone to WiredTiger </tutorial/change-standalone-wiredtiger/>` 
         guide in the Server manual.
```
